modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp

```diff
@@ -767,7 +767,11 @@ String organizationRoleSyncEntitiesEventName = liferayPortletResponse.getNamespa
 						document.<portlet:namespace />fm.<portlet:namespace />deleteRoleIds.value = <portlet:namespace />deleteRoleIds.join(',');
 					}
 
-					searchContainer.addRow(rowColumns, roleId);
+					var searchRow = searchContainer.addRow(rowColumns, roleId);
+
+					var searchRole = searchRow.one('td');
+
+					searchRole.addClass('table-cell-expand');
```

Comment:
You need to check if the DOM node you are looking up exists before you use it in JavaScript.